### ISSUE TITLE
Systemd service example file

### ISSUE THREAD

**Issue by chrissharp123** [kenstir/hemlock-sendmsg#1]
It will be useful to administrators running this service to have it start up, etc. alongside other services.  Below is an example systemd unit file that assumes that the installation is to /usr/local/bin/hemlock-sendmsg/:

```
[Unit]
Description=a small daemon for sending push notifications to the Hemlock mobile apps
After=network.target

[Service]
ExecStart=/usr/local/bin/hemlock-sendmsg/hemlock-sendmsg
Type=exec
#Restart=on-failure

[Install]
WantedBy=default.target
```

The unit file would be moved to /etc/systemd/system/ and named hemlock-sendmsg.service.  This is currently running on a PINES testing server and works so far as expected.

EDIT: updated systemd file to comment out the Restart directive.

**Comment by kenstir** [comment]
Great idea.  Would it be enough to provide a `make install` target that copies a template to `/etc/systemd/system/hemlock-sendmsg.service` ?  I don't think I have the appetite to run an apt repository.  Alternatively I could use github actions to build binaries so that you wouldn't have to install Golang locally, but that seems like a small win.

**Comment by chrissharp123** [comment]
yes, a make install target makes sense to me - I don't think you'd need to run an APT repo or build debs

BTW, in my testing I see that the `Restart=on-failure` directive was causing trouble and the service unit works without it, so I've removed that line locally.

**Comment by kenstir** [comment]
Fixed in v1.1.0.  Let me know if that meets your needs.
